TPP.Core/Commands/Definitions/BadgeCommands.cs

```diff
@@ -134,15 +141,150 @@ public async Task<CommandResult> SelectBadge(CommandContext context)
 
         public async Task<CommandResult> Pokedex(CommandContext context)
         {
-            User user = (await context.ParseArgs<Optional<User>>()).OrElse(context.Message.User);
+            (Optional<User> optionalUser, Optional<string> mode_, Optional<User> optionalUser2) =
+                await context.ParseArgs<Optional<User>, Optional<string>,Optional<User>>();
+            User user = optionalUser.IsPresent ? optionalUser.Value : context.Message.User;
             bool isSelf = user == context.Message.User;
-            int numUniqueSpecies = (await _badgeRepo.CountByUserPerSpecies(user.Id)).Count;
-            return new CommandResult
+
+            // Have to additionally check if the string is not empty, otherwise "!dex " would jump right into the else - branch.
+            //   Not sure if this is an actual issue when connected to twitch
```

Comment:
This sounds like an issue I could fix in the argument parsing code by collapsing and trimming whitespace. I'll try that

Reply:
yep, wasn't too hard. See https://github.com/TwitchPlaysPokemon/tpp-core/pull/199

Reply:
Removed